Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/NativeInterface.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    public static class NativeInterface
```

Comment:
>public [](start = 4, length = 6)

for now I would suggest making this an internal class so that we don't need to provide documentation for the public functions below.